src/instruments/src/EFB/Settings/Settings.tsx

```diff
@@ -31,6 +32,7 @@ const PlaneSettings = () => {
     const [accelerationOutAltSetting, setAccelerationOutAltSetting] = useState(accelerationOutAlt);
     const [defaultBaro, setDefaultBaro] = usePersistentProperty('CONFIG_INIT_BARO_UNIT', 'IN HG');
     const [weightUnit, setWeightUnit] = usePersistentProperty('CONFIG_USING_METRIC_UNIT', '1');
+    const notification = new Notification();
```

Comment:
I'm not sure if instantiating those objects every render is a good idea.

Maybe store them in a `useState(() => new Notification())`, or use them at the call site (probably the latter).